res/img/element-icons/maximise_expand.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Hmm, please appease my OCD and use `-` for file name separators. That's the usual style for image file name, though I know their are outliers.